pb-crm-api/src/db/entities/email.entity.ts

```diff
@@ -10,17 +14,25 @@ export class EmailEntity implements EmailModel {
     @PrimaryGeneratedColumn('uuid')
     id: string
 
-    @Column('varchar', { nullable: false })
+    @Column({
+        type: "character varying",
```

Comment:
what's the difference? doesnt this mean the same as `(varchar, {nullable: false})`?

Reply:
Yeah I misread the documentation, it is the same thing. A varying length, non-padded string with a hard limit (truncated by sql without error)